core/assets/bundles/bundle_fr.properties

```diff
@@ -466,9 +487,11 @@ play = Jouer
 campaign = Campagne
 load = Charger
 save = Sauvegarder
-fps = FPS: {0}
+fps = IPS: {0}
```

Comment:
FPS should not be translated, IPS is a term that is never used and the average french player knows what FPS means.